load.Rmd

```diff
@@ -355,7 +359,7 @@ logit_p <- -3.2941 + (-0.4659 * l$dist_fast) + (1.3430 * dist_fastsqrt) + (0.006
 # + (2.4783 * dutch) + (-0.0706 * dutch*dist_fast) + (0.0286 * ebike) + (0.0902 * ebike*dist_fast) + (-0.0014 * ebike*dist_fastsq)
 
 l$slc <- exp(logit_p) / (1 + exp(logit_p))
-cormod <- cor(flow$clc, l$slc) # crude indication of goodness-of-fit
+cormod <- cor(l$clc, l$slc) # crude indication of goodness-of-fit
```

Comment:
Why have we got from using `flow` to `l`?  `flow` is directly from the census and `l` is flow mapped to the route network.

I can see in the rest of the file we use `l@data` but wasn't sure why.